server/@types/bapv.d.ts

```diff
@@ -191,4 +191,5 @@ export type VisitSessionData = {
     phoneNumber: string
     contactName?: string
   }
+  reservationId?: number
```

Comment:
Do we also need to store the status (i.e. `RESERVED` or `BOOKED`)? E.g. `reservation { id: number, status: 'RESERVED' | 'BOOKED' }`. May well not need to; I suppose it depends what we need at the final stages...